integrated_channels/canvas/migrations/0001_initial.py

```diff
@@ -44,8 +44,8 @@ class Migration(migrations.Migration):
             ],
             options={
                 'get_latest_by': 'history_date',
-                'ordering': ('-history_date', '-history_id'),
                 'verbose_name': 'historical canvas enterprise customer configuration',
+                'ordering': ('-history_date', '-history_id'),
```

Comment:
fairly inconsequential, but does the reordering matter here?

Reply:
this was automatically generated.. no idea what this means .. let's see what the latest migrations say

Reply:
I assumed it was an isort issue.